util/dvsim/SimCfg.py

```diff
@@ -112,6 +112,12 @@ def __init__(self, flow_cfg_file, proj_root, args):
         self.dry_run = args.dry_run
         self.map_full_testplan = args.map_full_testplan
 
+        # Choose a value for dv_root. This is currently always at
+        # <proj_root>/hw/dv but at least listing it here rather than in the
+        # various hjson files means that other projects vendoring us in can
+        # patch the path simply.
+        self.dv_root = os.path.join(proj_root, 'hw', 'dv')
```

Comment:
The tool should make no assumptions on what paths exist. Please remove these lines. Instead add this to `hw/data/common_project_cfg.hjson`. I would imagine the Ibex repo will provide its own 'project' specific hjson that sets the `dv_root` correctly as needed.

Reply:
The problem is that `common_project_cfg.hjson` is included by `common_sim_cfg.hjson`, which is explicitly named in each IP's hjson file and is found relative to `dv_root`. It feels a little ugly to replace `{proj_root}/hw/dv` with `{dv_root}`, except if it happens to be the inclusion of `common_sim_cfg.hjson`.

Is that what you would like to do? Or can you think of another trick?